UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -100,5 +101,58 @@ public void WhenInValidRequest_ThenDownloadFileShareServiceFilesReturnsNoFile()
             var result = fulfilmentFileShareService.DownloadFileShareServiceFiles(message, fulfilmentDataResponses, "");
             Assert.IsNotNull(result);
         }
+
+        [Test]
+        public async Task WhenRequestSearchReadMeFilePath_ThenReturnsFilePath()
+        {
+            string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
+            string exchangeSetRootPath = @"D:\\Downloads";
+
+            A.CallTo(() => fakefileShareService.SearchReadMeFilePath(A<string>.Ignored)).Returns(exchangeSetRootPath);
+            var result = await fulfilmentFileShareService.SearchReadMeFilePath(batchId);
+
+            Assert.IsNotNull(result);
+            Assert.AreEqual("ReadMe Text file path found", "ReadMe Text file path found");
+        }
+         [Test]
+        public async Task WhenRequestSearchReadMeFilePath_ThenReturnsEmptyStringResponse()
```

Comment:
Is the naming on this test correct? 

Should the first part of the name infer the reason for the test as I'm not sure that it currently does?

Reply:
Modified.